plugins/modules/web_app_pool_info.ps1

```diff
@@ -0,0 +1,96 @@
+#!powershell
+
+# Copyright: (c) 2024, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+    options = @{
+        name = @{ type = "str" }
+    }
+    supports_check_mode = $true
+}
+
+$module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
+
+$name = $module.Params.name
+
+$module.Result.exists = $false
+$module.Result.appPools = @()
```

Comment:
We try and keep the `snake_case` format for any input/output values in a module.

Reply:
https://github.com/ansible-collections/microsoft.iis/commit/49e95c9684ce9ce31dba2a03438eda006795c927